### DISCUSSION TITLE
AttributeError

### DISCUSSION THREAD

**Discussion by leilamalihi** [yoshitomo-matsubara/torchdistill#336] [Q&A]
I have this Error in line 250, -model_t.eval(), which is:
line 250, in main_worker
    model_t.eval()

AttributeError: 'dict' object has no attribute 'eval'

Can you please help me to solve that?

**Comment by yoshitomo-matsubara** [comment]
Hi @leilamalihi 

The current description doesn't give me much context.

Please
1. explain what script you found model as a dict at Line 250
2. provide the edited code AND config files
3. clarify what you wanted to do (but failed to)
4. share the version of torchdistill
